node.go

```diff
@@ -20,6 +20,7 @@ import (
 	"time"
 
 	"github.com/fatih/color"
+	"golang.org/x/text/unicode/norm"
```

Comment:
You will have to run `dep ensure -add golang.org/x/text/unicode/norm` and update the `Gopkg.toml` and lockfile, and commit the vendored library as well.